serde/src/java/org/apache/hadoop/hive/serde2/lazy/fast/LazySimpleDeserializeRead.java

```diff
@@ -350,7 +350,12 @@ public LazySimpleDeserializeRead(TypeInfo[] typeInfos,
       throw new RuntimeException("serialization.last.column.takes.rest not supported");
     }
 
-    timestampParser = new TimestampParser();
+    List<String> timestampFormats = lazyParams.getTimestampFormats();
+    if (timestampFormats != null) {
```

Comment:
minor: i would use ternary operator here

Reply:
Thanks for the review @deniskuzZ.
I updated the code with ternary operator.